content/getting-started/sections/installing-atom.md

```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
 
 #### Installing Atom on Linux
 
-To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. These packages do not currently have auto-update features, so when you would like to upgrade to a new release of Atom, you will have to repeat this installation process.
+To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. If you'd like Atom to be updated when new releases are published, you can configure an official package repository for your distribution.
 
 [releases]: https://github.com/atom/atom/releases/latest
 
 ##### Debian and Ubuntu (deb/apt)
 
-To install Atom on Debian, Ubuntu, or related systems:
+To install Atom on Debian, Ubuntu, or related distributions, add our official
+package repository to your system by running the following commands:
+
+```
+curl -L https://packagecloud.io/AtomEditor/atom/gpgkey | sudo apt-key add -
+sudo sh -c 'echo "deb [arch=amd64] deb https://packagecloud.io/AtomEditor/atom/any/ any main" > /etc/apt/sources.list.d/atom.list'
+sudo apt-get update
+```
+
+You can now install Atom using `apt-get`:
+
+```
+# Install Atom
+sudo apt-get install atom
+
+# Install Atom Beta
+sudo apt-get install atom-beta
+```
+
+You can also download the Atom .deb package and install it directly:
```

Comment:
Should I say "Alternatively, you can..."?

Reply:
Yes, that works :+1: